### PR TITLE
Clarify module compatibility with Prestashop and php version

### PR DESCRIPTION
**Pull request by nenes25**
Add checks about compatibility with Prestahop and php version in module README
( Fix wrong licence in readme too )

### CODE REVIEW

**Review by SebSept — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 This repository provides a Command that better supports PrestaShop legacy classes using the current PrestaShop Console and a list
 of useful commands that you can use and reuse for learning purposes.
 
-This module is released under MIT license.
+This module is released under AFL license.
```
Comment (SebSept) [inline README.md:6]: to be exact AFL 3.0
But that's ok.
thanks for the catch :)

**Comment by jf-viguier** [comment]
Ok Thanks

**Comment by nenes25** [comment]
ping @jf-viguier can we merge this ?

**Comment by jf-viguier** [comment]
@nenes25 have-you tested with php 7.3 ?

**Comment by nenes25** [comment]
@jf-viguier the test are only done on syntax through github actions cf. https://github.com/friends-of-presta/fop_console/runs/1541729611?check_suite_focus=true ( by the way we could fix name and add more versions if needed i'll check)
But i have not make manual testing.
I can remove the check until then if you want ?

**Comment by jf-viguier** [comment]
@nenes25 I've just tested with php 7.3 and PRestashop 1.7.7 : all command work, can you check ps 1.7.7 ?

**Review by jf-viguier — APPROVED** [review]
ok for me

**Pull request merged**